recipes/raxml/build.sh

```diff
@@ -24,8 +33,8 @@ for PTHREADS in "" .PTHREADS; do
     else
       MAKEFILE=${MAKEFILE}.gcc
     fi
-    make -f ${MAKEFILE} CC=$CC
```

Comment:
Why did you remove `CC=$CC` ?
I think this is the reason why now you need the `patch` (to remove `CC = gcc`.

Reply:
🤦 stupid me, thanks for noticing!